rllib/BUILD

```diff
@@ -671,6 +671,17 @@ py_test(
 )
 
 # PPO
+
```

Comment:
Let's add this test to the learning tests:
Reason: We reference pong-ppo in many places in our docs and examples.
It's a pretty foundational test so let's include it in CI.